client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -20,7 +20,8 @@ import SimpleNotice from 'notices/simple-notice';
 const CancelPrivateRegistration = React.createClass( {
 	getInitialState() {
 		return {
-			disabled: false
+			disabled: false,
+			cancelling: false
```

Comment:
Not our own, but the React docs recommend that you use the [minimal representation of UI state](https://facebook.github.io/react/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state). Keeping state as small as possible also means that we'll have fewer possible states to test (even though this component can currently never get in a state where canceling !== disabled). We follow this principle enough in Calypso that two separate properties with the same value in state might be confusing.